src/pages/iou/IOUModal.js

```diff
@@ -43,29 +64,43 @@ const Steps = {
     IOUConfirm: 'Confirm',
 };
 
-// The steps to be shown within the create IOU flow.
-const steps = [Steps.IOUAmount, Steps.IOUParticipants, Steps.IOUConfirm];
-
 class IOUModal extends Component {
     constructor(props) {
         super(props);
-
         this.navigateToPreviousStep = this.navigateToPreviousStep.bind(this);
         this.navigateToNextStep = this.navigateToNextStep.bind(this);
         this.currencySelected = this.currencySelected.bind(this);
+        this.addParticipants = this.addParticipants.bind(this);
         this.createTransaction = this.createTransaction.bind(this);
         this.updateComment = this.updateComment.bind(this);
         this.addParticipants = this.addParticipants.bind(this);
+        const participants = lodashGet(props, 'report.participants', []);
+        const participantsWithDetails = getPersonalDetailsForLogins(participants, props.personalDetails)
+            .map(personalDetails => ({
+                login: personalDetails.login,
+                text: personalDetails.displayName,
+                alternateText: personalDetails.login,
+                icons: [personalDetails.avatar],
```

Comment:
There should now be an avatarThumbnail on personalDetails which is an 80x80 version of the image, can we double check to make sure we're using those instead of full sized? You might also need to update getPersonalDetailsForLogins

Reply:
Good spot @thienlnam. 

I could be wrong, but I think that might be out of scope for this issue as participant Avatars were implemented in a previous issue. If we spot that the incorrect size is used though, I'll be happy to create a new issue for this.

Reply:
That's fair, let's go ahead with that then if the incorrect sizes are showing up